node/runtime/src/lib.rs

```diff
@@ -62,7 +62,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	impl_name: create_runtime_str!("substrate-node"),
 	authoring_version: 10,
 	spec_version: 35,
-	impl_version: 37,
+	impl_version: 38,
```

Comment:
Please revert, you only changed comments and nothing more.

Reply:
Yeah, but then the CI always fails because I changed these files without updating the `impl_version`, we introduced this checks some weeks ago. It's unfortunate, but I think there's currently no way around it without making the CI check more complicated (i.e. excluding comments).

Reply:
This can be ignored. A non-required check is failing.